### ISSUE TITLE
learning_rate_schedule error when passing a list of values

### ISSUE THREAD

**Issue by jcsharp** [Microsoft/CNTK#1109]
When I execute this in Python:

`lr = [0.0005, 0.00025, 0.00005]`
`learning_rate_schedule(lr, UnitType.sample, epoch_size=5000)`

I get:

`learning_rate_schedule() got multiple values for argument 'epoch_size'`

How can I specify a customized learning rate schedule?

**Comment by raaaar** [comment]
Works fine with beta 5, what version do you have (`pip show cntk`)?

**Comment by jcsharp** [comment]
I have to stick with beta 3 for now, because I had even bigger problems with beta 5...

**Comment by wolfma61** [comment]
can you share those problems?

**Comment by jcsharp** [comment]
I gave beta 5 another try. I found out that I can get it to work with my reinforcement learning algorithm if I use a learning rate 100 times larger than the value that worked well with beta 3. A little odd, but I can live with that.

Now, when I use more than one value for the learning rate schedule, it does not work as I expect. It seems that the effective learning rate is always the lowest value (the last one). 

Here's what I tried:

`lr = [0.01, 0.0001, 0.0001]`
`lr_schedule = learning_rate_schedule(lr, UnitType.minibatch, epoch_size=5000)`
`learner = sgd(model.parameters, lr_schedule, gradient_clipping_threshold_per_sample=10)`
`trainer = Trainer(model, loss, eval_function, learner)`

Then I train the model with minibatches of 1024 samples:

`arguments = dict(zip(loss.arguments, [y, x]))`
`trainer.train_minibatch(arguments, outputs=[loss.output])`

I'm expecting my model to make good progress through the first 5000 minibatches, using 0.01 as the learning rate. After 5000 minibatches, it should make very little progress. However, I see no progress at all.

Am I doing something wrong?
